trek.js

```diff
@@ -0,0 +1,108 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+// LOAD TRIPS
+const loadTrips = () => {
+  const tripsList = $('#tbody');
+  tripsList.empty();
+
+  reportStatus('Loading trips! Please wait...');
+  $('#table').show();
+
+  axios.get(URL)
+  .then((response) => {
+    response.data.forEach((trip) => {
+      tripsList.append(`<tr id="${trip.id}"><td>${trip.id}</td>
+      <td>${trip.name}</td></tr>`);
+    });
+    reportStatus('Trips Loaded!');
+  })
+  .catch((error) => {
+    reportStatus('Error: ${error.message}');
+  });
+};
+
+// LOAD ONE TRIP
+
+const loadTrip = (id) => {
+        const tripInfo = $('#trip-info');
+        tripInfo.empty();
+        const reservationForm = $('#reservation-form');
+        reservationForm.empty();
+
+        reportStatus('Loading trip info! Please wait...');
+        $('#details').show();
+
+  // GET TRIP DETAILS FROM API
+
+        axios.get(URL + `/${id}`)
+          .then((response) => {
+    // console.log(response);
+          tripInfo.append(`<tr><td><strong>Name: </strong>     ${response.data.name}</td></tr>
+      <tr><td><strong>Trip ID: </strong>${response.data.id}</td></tr>
+      <tr><td><strong>Continent: </strong>${response.data.continent}</td></tr>
+      <tr><td><strong>Category: </strong>${response.data.category}</td></tr>
+      <tr><td><strong>Weeks: </strong>${response.data.weeks}</td></tr>
+      <tr><td><strong>Cost: </strong>${response.data.cost.toFixed(2)}</td></tr>
+      <tr><td><strong>About: </strong></br>${response.data.about}</td></tr>`);
+          reportStatus('Trip Info Loaded!');
+          reservationForm.append(`<div >
+        <label for="name">Trip:${response.data.name}</label>
+        </div>`);
+          reservationForm.append(`<div>
+          <label for="name">Name: </label>
+          <input type="text" name="user-name" />
+          </div>
+          <div>
+          <label for="email">Email: </label>
+          <input type="text" name="email" />
+          </div>
+          <div class="submit-container">
+          <input type="submit" name="add-reservation" value="Reserve" class="reserve" id="reserve${response.data.id}"/>
```

Comment:
Out of curiosity, why do you make the `id="reserve..."`? Why not just `id="${response.data.id}"`? Because of this you need to add the `.substr(7)` part on line 105